playback/src/audio_backend/alsa.rs

```diff
@@ -220,6 +222,7 @@ impl Sink for AlsaSink {
                 )
             })?
         }
+        self.period_buffer.clear();
```

Comment:
Is there any situation where we'd `write_buf()` _without_ `self.period_buffer.clear()`? Otherwise you could move this to `write_buf()`.

Reply:
Not that I can think of? I think we could do that.

Reply:
Yep, moving `self.period_buffer.clear()` to `write_buf()` works as you'd expect.

Reply:
Done.